fireant/database/data_blending.py

```diff
@@ -0,0 +1,29 @@
+from .type_engine import make_columns
+
+
+class DataBlender:
+    def __init__(self, primary_db, primary_type_engine, secondary_db, secondary_type_engine):
+        self.primary_db = primary_db
+        self.primary_type_engine = primary_type_engine
+        self.secondary_db = secondary_db
+        self.secondary_type_engine = secondary_type_engine
+
+    def derive_columns_from_secondary(self, secondary_schema, secondary_table):
```

Comment:
i think it would be an easier interface to understand if it was something like `DatabaseTypeTransformer.transform_columns(columns, from_db, to_db)` and separate the getting the columns bit out into another scope